app/screens/AuthLoading.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import React, { Component } from "react";
+import { AsyncStorage } from "react-native";
+
+import { Pending } from "components";
+import type { EmailType, navigationType, PasswordType } from "types";
+
+type Props = {
+  navigation: navigationType,
+  email: EmailType,
+  password: PasswordType,
+};
+
+class AuthLoading extends Component<Props> {
+  static navigationOptions = {
+    header: null,
+    title: "Login",
+  };
+
+  constructor(props: Props) {
+    super(props);
+    this.asyncLogin();
+  }
+
+  asyncLogin = async () => {
```

Comment:
It doesn't _seem_ like there's a reason to use this style of method definition in this case. How about the standard:

```js
async asyncLogin() {
  ...
}
```

Out of interest, is prepending async methods/functions with `async` a pattern you're experimenting with? Or is this a one off?